js/src/data/resolvers.js

```diff
@@ -350,3 +354,52 @@ export function* getMappingSources( attributeKey ) {
 		);
 	}
 }
+
+/**
+ * Fetches the Attribute Mapping Rules and calls receive action
+ *
+ * @param {Object} pagination Object containing client pagination parameters like page and per_page
+ * @see AttributeMappingRulesController.php
+ */
+export function* getMappingRules( pagination ) {
+	try {
+		const response = yield fetchWithHeaders( {
+			path: addQueryArgs( `${ API_NAMESPACE }/mc/mapping/rules`, {
+				page: pagination.page,
+				per_page: pagination.perPage,
+			} ),
+			parse: false,
```

Comment:
`fetchWithHeaders` will directly specify/override the `parse` option, which probably doesn't need to be specified here.